app/Http/Transformers/UsersTransformer.php

```diff
@@ -30,7 +30,7 @@ public function transformUser(User $user)
                 'username' => e($user->username),
                 'remote' => ($user->remote == '1') ? true : false,
                 'locale' => ($user->locale) ? e($user->locale) : null,
-                'employee_num' => e($user->employee_num),
+                'employee_num' => ($user->employee_num) ? e($user->employee_num) : null,
```

Comment:
This feels a little unrelated but I guess it's fine.

Reply:
It is, but it was wrong before. :(